packages/legacy/core/App/contexts/network.tsx

```diff
@@ -41,13 +41,15 @@ export const NetworkProvider: React.FC<React.PropsWithChildren> = ({ children })
   }
 
   const assertLedgerConnectivity = async (): Promise<boolean> => {
-    const nodes = fetchLedgerNodes()
+    const hostname = hostnameFromURL(Config.MEDIATOR_URL!)
 
-    if (typeof nodes === 'undefined' || nodes.length === 0) {
+    if (hostname === null || hostname.length === 0) {
       return false
     }
 
-    const connections = await Promise.all(nodes.map((n: { host: string; port: number }) => canConnectToLedgerNode(n)))
+    const nodes = [{ host: hostname, port: 443 }]
```

Comment:
Do we need this array and Promise.all and includes check if we are just checking one value?

Reply:
No but it allows others to add hosts and they'll get magically checked also. We just happen to only care about one.